lib/i2cscan/i2cscan.cpp

```diff
@@ -6,7 +6,7 @@ uint8_t portArray[] = {16, 5, 4, 2, 14, 12, 13};
 uint8_t portExclude[] = {LED_PIN};
 String portMap[] = {"D0", "D1", "D2", "D4", "D5", "D6", "D7"};
 // ESP32C3 has not as many ports as the ESP32
-#elif defined(ESP32C3)
+#elif defined(ESP32C3) || defined(ESP32C6)
```

Comment:
Pleas check the portArray and port Exclude, i belive they have to be different from ESP32-C3

Reply:
Added custom PortMap for ESP32C6 and included all usable pins apart from serial TX/RX and USB D+/D-